backend/src/main/scala/com/softwaremill/adopttapir/starter/content/ContentMerger.scala

```diff
@@ -0,0 +1,45 @@
+package com.softwaremill.adopttapir.starter.content
+
+object ContentMerger {
```

Comment:
Maybe simply `DirectoryMerger` and an `apply` method? It's not quite clear what `create` does in an object called `Merger` :)